src/class/ServerManager/ServerManager.cpp

```diff
@@ -169,11 +169,27 @@ ServerManager::ServerManager(const IConf *conf)
 	std::list< const IServerConf * >::iterator end = servers.end();
 	for (; it != end; it++)
 	{
-		this->_virtual_servers.push_back(new Server(*it));
+		Server *new_server;
+		try
+		{
+			new_server = new Server(*it);
```

Comment:
same for virtual_servers being Server*

Reply:
Yeah, it was unnecessary for Server*. I fixed it

Reply:
Okay, yes, it was not necessary for Server to be a *